api/src/main/java/com/github/eyefloaters/console/api/mapping/InvalidPageCursorExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.github.eyefloaters.console.api.mapping;
+
+import java.util.List;
+
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.ExceptionMapper;
+import jakarta.ws.rs.ext.Provider;
+
+import org.jboss.logging.Logger;
+
+import com.github.eyefloaters.console.api.model.Error;
+import com.github.eyefloaters.console.api.model.ErrorResponse;
+import com.github.eyefloaters.console.api.support.ErrorCategory;
+import com.github.eyefloaters.console.api.support.InvalidPageCursorException;
+
+@Provider
+public class InvalidPageCursorExceptionHandler implements ExceptionMapper<InvalidPageCursorException> {
+
+    Logger logger = Logger.getLogger("com.github.eyefloaters.console.api.errors.client");
```

Comment:
Looks intentional but at the same time a bit unusual so just to double check - it is intentional that it's a string here and not this class's FQDN.

Reply:
The idea here is that all client errors can be configured with a common logger name (there is an equivalent for the 5XX server errors). An alternative would be to move the mappers into sub-packages, which I am not opposed to either.

Let's do it in a separate PR if it seems like a good idea. Minimally, a constant field with the logger names would probably be a good idea though.